STAT302 - assignment 2 Q 3.22

In [1]:
# Q 3.22 a
R2 = 9350 / 17190
R2

[1] 0.5439209

54.39% of the variability of calories can be explained by regression based on sugar and fiber.

In [2]:
# Q 3.22 b
sigmaE = sqrt(7840 / (36 - 2 -1))
sigmaE

[1] 15.41349

# Q 3.22 c
## ANOVA Table
|     |df|ss  |MS     |F      | \
|model|2 |9350|9350/2 |9350/2 | \
|error|33|7840|7840/33|7840/33|

In [3]:
f = (9350 / 2) / (7840 / 33)
f

[1] 19.67793

# Q 3.22 d
p-value = 0.00002, it's a small number. \
Represents the overall model is significant. \
At least sugars and/or fiber are related to calories.